draft-ietf-moq-transport.md

```diff
@@ -966,12 +988,18 @@ When a relay receives an incoming ANNOUNCE for a given namespace, for
 each active upstream subscription that matches that namespace, it SHOULD send a
 SUBSCRIBE to the publisher that sent the ANNOUNCE.
 
-Object headers carry a short hop-by-hop `Track Alias` that maps to
-the Full Track Name (see {{message-subscribe-ok}}). Relays use the
-`Track Alias` of an incoming Object to identify its track and find
-the active subscribers for that track. Relays MUST forward Objects to
-matching subscribers in accordance to each subscription's priority, group order,
-and delivery timeout.
+If a relay receives a Publisher-Chosen Subscription ID in SUBSCRIBE_OK, it
+SHOULD assign this ID to downstream subscriptions for the same track. Since
+subscribers can request tracks from uncoordinated publishers through a single
+relay session, it is not always possible to reuse the upstream PSID.  If there
+is no upstream Publisher-Chosen Subscription ID, or if the upstream PSID is
+already in use downstream for a different track, the relay SHOULD NOT set a
```

Comment:
may be it will help clarify further on where the relay SHOULD NOT set ? From what I understand, it is set on the downstream subscription?

Reply:
I think the conditions are pretty clear:

1. There is no upstream Publisher-Chosen Subscription ID
2. The upstream PSID is already in use downstream for a different track